CodeAssignment_MVVM_Combine/Sources/Controllers/DetailViewController.swift

```diff
@@ -0,0 +1,309 @@
+    }()
+    
+    /// Userのプロフィール画像
+    private lazy var userImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.image = UIImage(systemName: "person.circle")?.withTintColor(.systemMint, renderingMode: .alwaysOriginal)
+        imageView.clipsToBounds = true
+        imageView.layer.cornerRadius = 8
+        return imageView
+    }()
+    
+    /// User名を表示するためのlabel
+    private lazy var userNameLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.font = .systemFont(ofSize: 22, weight: .bold)
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    /// 当該リポジトリのStarの数も一緒に表示するStarボタン
+    private lazy var starButton: UIButton = {
+        var config = UIButton.Configuration.filled()
+        let image = UIImage(systemName: "star")?.withTintColor(.systemGray3, renderingMode: .alwaysOriginal)
+        if let image {
+            config.image = image.withConfiguration(UIImage.SymbolConfiguration(pointSize: 25, weight: .regular))
+        }
+        config.baseBackgroundColor = .white
+        config.imagePlacement = .leading
+        // imageとtext間のSpace
+        config.imagePadding = 8
+        config.contentInsets = .init(top: 10, leading: 8, bottom: 10, trailing: 8)
+        config.cornerStyle = .large
+        
+        // borderLayerの設定
+        var backgroundConfig = UIBackgroundConfiguration.clear()
+        backgroundConfig.strokeColor = .systemGray2
+        backgroundConfig.strokeWidth = 2
+        config.background = backgroundConfig
+        
+        let button = UIButton(configuration: config)
+        // content hugging priorityを設定し、buttonのconstraintsが意図通りに設定されないことを防ぐ
+        button.setContentHuggingPriority(.required, for: .horizontal)
+        button.setContentCompressionResistancePriority(.required, for: .horizontal)
+        button.titleLabel?.setContentHuggingPriority(.required, for: .horizontal)
+        button.titleLabel?.setContentCompressionResistancePriority(.required, for: .horizontal)
+        
+        return button
+    }()
+    
+    /** 言語ごとに色をつけて表示させるためのView
+    - starButtonをUIButton.Configurationを用いた作成に変えた後、layoutSubViewsでcornerRadiusを設定しようとしたら、反映されなかった。
+    - そのため、インスタンス生成時にcornerRadiusをするようにした
+     */
+    private lazy var languageColorView: UIView = {
+        let view = UIView()
+        view.clipsToBounds = true
+        view.layer.cornerRadius = Const.colorViewHeight / 2.0
+        view.backgroundColor = .systemPink
+        return view
+    }()
+    
+    private lazy var languageNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 20, weight: .regular)
+        label.numberOfLines = 0
+        // byCharWrapping: 単語ごとじゃなく、一文字ごとに改行する
+        label.lineBreakMode = .byCharWrapping
+        label.textColor = .black.withAlphaComponent(0.8)
+        label.setContentHuggingPriority(.init(999), for: .horizontal)
```

Comment:
マジックナンバーはつくらないほうがいいですね
```suggestion
        label.setContentHuggingPriority(.required, for: .horizontal)
```

Reply:
[7c2bd26]にてコード修正しました！ご確認お願いします！